core/iwasm/aot/aot_loader.c

```diff
@@ -211,6 +289,10 @@ loader_malloc(uint64 size, char *error_buf, uint32 error_buf_size)
 
 static char *
 const_str_set_insert(const uint8 *str, int32 len, AOTModule *module,
+#if defined(BUILD_TARGET_XTENSA) && (WASM_ENABLE_AOT_LOAD_FROM_VRAM != 0)
+                     bool is_vram_align_word,
+#endif /* end of defined(BUILD_TARGET_XTENSA) &&  \
+          (WASM_ENABLE_AOT_LOAD_FROM_VRAM != 0) */
```

Comment:
No need to add comments if the `#else/#endif` is very close to `#if`, and the code is easy to read.